CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+# 0.36.1 / 2020-04-02
+
+### Changes
+* [IMPROVEMENT] Connection timeout is now configurable [#289][]
+* [IMPROVEMENT] Fix json service check status [#287][]
+* [BUGFIX] Revert pull request [#285][] [#288][]
```

Comment:
nit: what's the user impact of this revert + PR #289? (might be worth merging the 2 items into 1 for clarity)